deployment-templates/compute-engine/forseti-instance.py

```diff
@@ -144,8 +146,7 @@ def GenerateConfig(context):
 
 # Install Forseti Security
 cd $USER_HOME
-rm -rf forseti-*
-rm -rf run_forseti.sh
+rm -rf *forseti*
```

Comment:
nit: Is the leading wildcard necessary?  From the above directory naming, would seem not.  I am just a bit worried about the potential to remove other directories with *forseti* in the name.